docs/new_router.rst

```diff
@@ -81,4 +81,4 @@ shortcut for::
     return route
 
 ``app.router.register_route(...)`` is still supported, it creates
-:class:`aiohttp.web.ResourceAdapter` for every call (but it's deprecated now).
+:class:`aiohttp.web.StaticResource` for every call (but it's deprecated now).
```

Comment:
This is a historic document pointing at something that doesn't exist anymore. So just remove the class role instead of pointing at a wrong thing.
```suggestion
``aiohttp.web.ResourceAdapter`` for every call (but it's deprecated now).
```

Reply:
I decided that `ResourceAdaptep` was replaced by `StaticResource`, but okay